util/update_bundled_ca_certificates.rb

```diff
@@ -4,6 +4,7 @@
 
 URIS = [
   URI('https://rubygems.org'),
+  URI('https://index.rubygems.org'),
   URI('https://staging.rubygems.org'),
   URI('https://s3.amazonaws.com'),
   URI('https://fastly.rubygems.org'),
```

Comment:
Added in the following commit: https://github.com/rubygems/rubygems/commit/ec6a1ce416f3f73c2d97b68f8d41f6867f985b9a

I think we're planning on accessing/hitting it soon for the new index? Fastly isn't mentioned anywhere specifically in the Rubygems codebase at the moment.